### ISSUE TITLE
Severity level for "lookup table reloaded from file" messages set to 3

### ISSUE THREAD

**Issue by atc0005** [rsyslog/rsyslog#2713]
### Expected behavior

Severity level of 6, *maybe* 5.

### Actual behavior

Severity level of 3 (error) for a condition that I believe justifies an informational level message.

From `/var/log/rsyslog.log`:

```
lookup table 'hostname_mappings' reloaded from file '/etc/hostname_mappings.json' [v8.34.0 try http://www.rsyslog.com/e/0 ]
```

When capturing the message using the debug template this is what I found:

```
msg: ' lookup table 'hostname_mappings' reloaded from file '/etc/hostname_mappings.json' [v8.35.0 try http://www.rsyslog.com/e/0 ]'
escaped msg: ' lookup table 'hostname_mappings' reloaded from file '/etc/hostname_mappings.json' [v8.35.0 try http://www.rsyslog.com/e/0 ]'
inputname: imuxsock rawmsg: '<43>May 15 21:44:42 rsyslogd: lookup table 'hostname_mappings' reloaded from file '/etc/hostname_mappings.json' [v8.35.0 try http://www.rsyslog.com/e/0 ]'
```

### Environment
- rsyslog version: 8.35.0
- platform: Ubuntu 16.04

**Comment by davidelang** [comment]
I agree that error is too high a level for a routine reload where everything 
works.

**Comment by davidelang** [comment]
Adam, do you want to create a PR to change this message?

**Comment by atc0005** [comment]
> Adam, do you want to create a PR to change this message?

Hi @davidelang,

I briefly looked at the lookup.c file and found where the message is being emitted:

```c
finalize_it:
	if (iRet != RS_RET_OK) {
		if (stub_val == NULL) {
			LogError(0, RS_RET_INTERNAL_ERROR,
					"lookup table '%s' could not be reloaded from file '%s'",
					pThis->name, pThis->filename);
		} else {
			LogError(0, RS_RET_INTERNAL_ERROR,
					"lookup table '%s' could not be stubbed with value '%s'",
					pThis->name, stub_val);
		}
		lookupDestruct(newlu);
	} else {
		if (stub_val == NULL) {
			LogError(0, RS_RET_OK, "lookup table '%s' reloaded from file '%s'",
					pThis->name, pThis->filename);
		} else {
			LogError(0, RS_RET_OK, "lookup table '%s' stubbed with value '%s'",
					pThis->name, stub_val);
		}
		lookupDestruct(oldlu);
	}
	RETiRet;
}
```

My initial impression is that the equivalent of "all is well" is being returned, which I would expect would be mapped to a different code, but evidently something else is causing the error code (vs informational message).

Unfortunately I've not worked with C in years and even then I never progressed past the initial intro course in college. Sorry, if I had a clue what I was doing here I'd gladly offer a PR.

**Comment by atc0005** [comment]
> so, it looks like this is the place where it's being generated, look at other
> messages that are being sent out, do they all use LogError, or is there a
> LogInfo or similar (I'd have to go digging to see where LogError is being
> defined)

Thanks, that's a good point; I guess I just assumed that there was only one variation of "LogSomething" that used an argument to identify the type of message it was handing.

I took a quick look and found that in addition to LogError there is also LogMsg. I'll try to dig further later and see what other variations exist.

**Comment by davidelang** [comment]
Thanks, while you are willing to keep digging, I'll keep advising/hinting. If 
you get to a point where you think this is too much work, let me know and I'll 
try to find time to do the same research (or it will sit until someone more 
familiar with the code gets time to do it)

**Comment by rgerhards** [comment]
2018-05-16 22:50 GMT+02:00 Adam Chalkley <notifications@github.com>:

> so, it looks like this is the place where it's being generated, look at
> other
> messages that are being sent out, do they all use LogError, or is there a
> LogInfo or similar (I'd have to go digging to see where LogError is being
> defined)
>
> Thanks, that's a good point; I guess I just assumed that there was only
> one variation of "LogSomething" that used an argument to identify the type
> of message it was handing.
>
> I took a quick look and found that in addition to LogError there is also
> LogMsg. I'll try to dig further later and see what other variations exist.
>

Thanks for trying to work on this. Do a

$ grep -r --include "*.c" LogMsg

on the source base directory. You are on the right path and what the grep
brings up should immediately make clear what can be changed ;-)

Rainer

**Comment by atc0005** [comment]
@davidelang @rgerhards 

Thanks for the encouragement and direction. I've submitted PR #2725 just now for review. Please let me know whether I'm anywhere close.

**Comment by rgerhards** [comment]
fixed via #2725
